2024/survey.yaml

```
@@ -356,6 +356,19 @@ questions:
       - Wiki (unofficial)
       - Blog posts
       - Others
+  - prompt: If you want to contribute or has contributed in the past to Nixpkgs, please select all that apply:
+    type: multiple
+    choices:
+      - I don't want/need to contribute.
```

make it less inquisitive
```suggestion
      - I don't need to contribute.
```